kanselarij-vlaanderen/frontend-kaleidos
# KAS-1591 - Des/enhancement/infotilerefactor

## What does this PR do?

Replaces info-tile component with block link component.

The info tile component was only used once on the mock login screen,  and never in the complexity and variants that the code had.

By replacing it with a simple block link component, we reduce the code complexity and move one step away from Flanders Gov components.

## Possible improvements

The HTML that comes out of this is a bit dirty.

```
<div id="ember358" class="ember-view">
   <div style="cursor: pointer">
      <div class="vlc-block-link">
         Admin Test
      </div>
   </div>
   <div style="cursor: pointer">
      <div class="vlc-block-link">
         Minister Test
      </div>
   </div>
   <div style="cursor: pointer">
      <div class="vlc-block-link">
         User Test
      </div>
   </div>
   <div style="cursor: pointer">
      <div class="vlc-block-link">
         Kanselarij Test
      </div>
   </div>
   <div style="cursor: pointer">
      <div class="vlc-block-link">
         Privileged Test
      </div>
   </div>
</div>
```

I would rather see markup like this.

```
<button class="vlc-block-link">Admin Test</button>
<button class="vlc-block-link">Minister Test</button>
<button class="vlc-block-link">User Test</button>
<button class="vlc-block-link">Kanselarij Test</button>
<button class="vlc-block-link">Privileged Test</button>
```

I tried to change this by adding a new mock-login.js component with an empty tagName (`tagName=""`), but then this was an old style ember component and I didn't know how to get it to work.

This PR is mergable without this enhancement.

If this enhancement is done, I will need to revise the styling.

## To test this

Go to http://localhost:4200/mock-login and check out the new look.

It should look similar the old look.
